### PR TITLE
Fixes to benchmark example

### PR DESCRIPTION
**Pull request by ibgreen**
Screen Shot 2020-04-02 at 4 04 36 PM

### CODE REVIEW

**Comment by ibgreen** [comment]
@heshan0131

**Comment by xintongxia** [comment]
do you have a screenshot?

**Comment by xintongxia** [comment]
it seems require polyfills when building the website.

**Comment by ibgreen** [comment]
> do you have a screenshot?

@xintongxia added. Took a bit of work to get these working in the website. Just need one more round of UI polish, will do in separate PR.

**Comment by xintongxia** [comment]
> > do you have a screenshot?
> 
> @xintongxia added. Took a bit of work to get these working in the website. Just need one more round of UI polish, will do in separate PR.

:) thanks for the efforts!

File: test/bench/modules.js [hunk 1]
```diff
@@ -18,36 +18,24 @@
...
+import ALIASES from '../../test/aliases';
+import {_addAliases} from '@loaders.gl/loader-utils';
 
+import imageBench from '@loaders.gl/images/test/images.bench';
 import coreBench from '@loaders.gl/core/test/core.bench';
 import csvBench from '@loaders.gl/csv/test/csv.bench';
 import jsonBench from '@loaders.gl/json/test/json-loader.bench';
-import dracoBench from '@loaders.gl/draco/test/draco.bench';
-import imageBench from '@loaders.gl/images/test/images.bench';
+// import dracoBench from '@loaders.gl/draco/test/draco.bench';
```
Comment (xintongxia) [inline test/bench/modules.js:30]: `dracoBench` doesn't work?

Comment (ibgreen) [reply test/bench/modules.js:30]: It works if you run the example separately, it doesn't work in the gatsby website. Module loading fails. 

I have spent too much time debugging how to make loader benchmarks work in gatsby to fix it right now :)

**Pull request merged**